root/release/edit_relationships.tt

```diff
@@ -17,6 +17,8 @@
 
       <p>[% l('To use the batch tools, select some recordings or works using the checkboxes.') %]</p>
 
+      <p>[% l('For more information, see {csg_relationships|relationship style guideline}.', { csg_relationships => { href => doc_link('Style/Relationships'), target => '_blank' } }) %]</p>
```

Comment:
```suggestion
      <p>[% l('Please read {relationships_doc|our guidelines for relationships} if you haven’t already.', { relationships_doc => { href => doc_link('Style/Relationships'), target => '_blank' } }) %]</p>
```

csg => classical style guide, it probably applied wherever you copied this from but not here :) 
Changed the text a bit to make it more clear that we expect people to follow those guidelines - let me know what you think :)